README.md

```diff
@@ -211,6 +211,14 @@ Every commit to your `main` branch will trigger a deployment to your production
 environment, and every commit to your `dev` branch will trigger a deployment to
 your staging environment.
 
+- **Create a Sentry account.** (Can be deferred to later)
+
+NOTE: this step is only required if you want to use Sentry for monitoring.
+
+Create a Sentry account with a Remix project [here](https://sentry.io/signup/?project_platform=javascript-remix)
+
+Copy the DSN from the code snippet to your `fly.toml` file in the `env` section and name it `SENTRY_DSN`. 
```

Comment:
Not sure how detailed we want the instructions to go. Note that the flow might not be super well optimized for this template because we show code snippets/instructions that's already there:

<img width="1101" alt="Screen Shot 2023-05-15 at 12 54 18 PM" src="https://github.com/epicweb-dev/epic-stack/assets/8533851/89fafb37-4630-4ca8-b1bd-9a06c9ad6a10">

If we wanted, Sentry could build out a flow that will take the user directly to the DSN page as shown below:


<img width="1050" alt="Screen Shot 2023-05-15 at 12 55 37 PM" src="https://github.com/epicweb-dev/epic-stack/assets/8533851/da3882c0-f7ff-4b5a-a5c8-8ea51dd89429">

Just lmk :)

Reply:
We should probably move everything to the docs eventually and have only basic instructions or just a link in the README. Would you mind adding a docs page for this and linking to that in this step to get us started?

Reply:
@kentcdodds when you say "move everything to docs", what parts are you referring to?

Reply:
I mean we should have a docs page for the setup process instead of sticking everything in the README.md. I'll make that change today probably so it's more clear for you where this should go.